configs/raspberrypi_config

```diff
@@ -49,7 +57,10 @@ function platform_modify() {
   # Ubuntu Server images actually use U-boot by default on RPi3 and the
   # layout is slightly different on the boot partition. That is why we need
   # additional logic here to determine what we are converting.
-  if [ -e work/boot/uboot.bin ] && [ -e work/boot/vmlinuz ]; then
+  if [ -e work/boot/uboot_rpi_4_32b.bin ] && [ -e work/boot/vmlinuz ]; then
```

Comment:
Will this not break the Rpi3 board configs?

Reply:
Good catch @oleorhagen. Also since you have the RPi4 board now, you can take over this PR :)

Reply:
haha, gee. thx!

Reply:
@oleorhagen no, because it ended up in the `elif`
Anyway, we tested this PR and it doesn't work, the board doesn't boot.

Reply:
> Will this not break the Rpi3 board configs?

Actually now that I looked, the code looks ok to me. The RPi3 functionality is left intact.

```
 if [ -e work/boot/uboot_rpi_4_32b.bin ] && [ -e work/boot/vmlinuz ]; then
    RASPBERRYPI_KERNEL_IMAGE="vmlinuz"
￼    RASPBERRYPI_BOOTLOADER_IMAGE="uboot_rpi_4_32b.bin"
￼elif [ -e work/boot/uboot.bin ] && [ -e work/boot/vmlinuz ]; then
    RASPBERRYPI_KERNEL_IMAGE="vmlinuz"
    RASPBERRYPI_BOOTLOADER_IMAGE="uboot.bin"
...
```

Reply:
lol, I missed the `elif` too. Sry